monitor/main.py

```diff
@@ -164,6 +162,8 @@ def collect(aws):
     else:
         session = boto3.session.Session(
             profile_name=args['aws_profile'], region_name=args['aws_region'])
+            #Loading additional AWS models
```

Comment:
I might be missing pieces of the puzzle, but aren't the new images based on Python 3.9 + new botocore, ie they don't need that code anymore (https://github.com/openshift/managed-prometheus-exporter-base/commit/92dabcd9ca384580b931c1631f807ddfb80c9f8d) ?

Reply:
You're correct, the updated images don't need the additional models. Thank you!